$NetBSD: patch-configure,v 1.1 2017/07/14 12:55:45 joerg Exp $

Remove hackaround for old OpenBSD bugs that break builds with other STL
implementations.

--- configure.orig	2017-07-13 11:13:06.550332512 +0000
+++ configure
@@ -19581,9 +19581,6 @@ SWIG_CXXFLAGS=
 if test yes = "$GXX" ; then
                     SWIG_CXXFLAGS="-fno-strict-aliasing"
 
-              XAPIAN_LIBS="$XAPIAN_LIBS -lstdc++"
-
-
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 #ifdef __INTEL_COMPILER
